alf/summary/summary_ops.py

```diff
@@ -147,6 +158,37 @@ def histogram(name, data, step=None, bins=None, walltime=None, max_bins=None):
         name, data, step, bins=bins, walltime=walltime, max_bins=max_bins)
 
 
+@_summary_wrapper
+def embedding(name, data, step=None, class_labels=None, label_imgs=None):
+    """Add embeddings to summary. The potentially high-dimensional embeddings
+    will be projected down to either 2D or 3D for visualization, with several
+    projection techniques to choose from in Tensorboard.
+
+    The visualized embeddings can be seen in the "PROJECTOR" page of Tensorboard.
+
+    Note: if this function is called multiple times, on the page there will be
+    multiple visualizations, each for every step.
```

Comment:
each for every step --> each for every call?

Reply:
updated